README.md

```diff
@@ -1,4 +1,34 @@
 # teslausb
+## Armbian orange pi zero 2 setup *tweaks
+
+apt pacakages:
+
+* xfsprogs
+* dos2unix
+* hping3
+* libxxhash0
+* autofs
+* python3-pip
+* nginx
+* fcgiwrap
+* libnginx-mod-http-fancyindex
+* libfuse-dev
+* exfatprogs
+* sntp
+
+first boot setup wifi with `nmtui`
+
+make a wpa_supplicant.conf at /etc/wpa_supplicant/wpa_supplicant.conf
+
+reboot
+
+to commit network changes
+
+now run the teslausb install
+
+```curl https://raw.githubusercontent.com/marcone/teslausb/main-dev/setup/generic/install.sh | bash```
```

Comment:
Honestly not sure what teslausb does with wpa_supplicant but armbian doesnt make one so i think its required but could run a fresh install to confirm. I know the teslausb script checks for wpa_supplicant and copies the template to etc at some point so it may be pointless if it overwrites the file at the start anyways.